source/common/http/utility.cc

```diff
@@ -1021,10 +1021,15 @@ void Utility::extractHostPathFromUri(const absl::string_view& uri, absl::string_
    *  pos:         ^
    *  host_pos:       ^
    *  path_pos:                       ^
+   *  scheme = "https"
    *  host = "example.com:8443"
    *  path = "/certs"
    */
+
+  // Find end of scheme
```

Comment:
style-nit: comments end with '.'

Reply:
ok I can add a '.' but there are other comments that do not have a '.' at the end. if this is a real style issue it should be enforced by a CI task.